base/applications/rapps/gui.cpp

```diff
@@ -1605,7 +1618,7 @@ class CMainWindow :
             break;
 
         case ID_ABOUT:
-            ShowAboutDialog();
+            ShowAboutDlg();
```

Comment:
Why rename the function?

Reply:
Because I copy / pasted the code pattern from @HBelusca PR.
Do you want me to keep the original function name from this module ?